### PR TITLE
Feature/integration

### PR DESCRIPTION
**Pull request by neotsai0622**
1. Added flag for sending result & targeted message so it does not try to send to 2nd device when single player mode
2. Removed checks to deny 2nd player for now
3. Added new lines at EOF
4. Changed print out to have better clarity of the message received
5. Changed variable i to j in the for loop

### CODE REVIEW

File: software/gamelogic/gamelogic.cpp [hunk 1]
```diff
@@ -57,9 +57,11 @@ int main () {
             }
         }
         else { // dealing with UI / BT
+            cout << "i'm here" << endl;
             Init_RS232();
```
Comment (paullin03) [inline software/gamelogic/gamelogic.cpp:61]: I want to confirm, these functions are "safe" if there is only one RFS chip?

File: software/gamelogic/gamelogic.cpp [hunk 2]
```diff
@@ -57,9 +57,11 @@ int main () {
             }
         }
         else { // dealing with UI / BT
+            cout << "i'm here" << endl;
             Init_RS232();
             Init_RS232_WIFI();
             blankscreen( BLUE );
+            cout << "past init" << endl;
             int player_num;
```
Comment (paullin03) [inline software/gamelogic/gamelogic.cpp]: I prefer num_players, player_num is a tad confusing

File: software/gamelogic/inputs.cpp [hunk 3]
```diff
@@ -167,30 +164,32 @@ int get_shoot_message_BT(shootvalues *input, int device_num) {
...
+void send_result_message_BT(int device_num, int x, int y, int gamestatus, int hitstatus, bool single_player_mode) {
     stringstream message;
     message << "result " << x << " " << y << " " << gamestatus << " " << hitstatus << "~";
 
     if (device_num == 1) {
         BT_send_0(message.str().c_str());
     }
     else {
-        BT_send_1(message.str().c_str());
+        if (!single_player_mode) {
```
Comment (paullin03) [inline software/gamelogic/inputs.cpp]: nit: else if instead

File: software/gamelogic/inputs.cpp [hunk 4]
```diff
@@ -200,15 +199,17 @@ void send_result_message_BT(int device_num, int x, int y, int gamestatus, int hi
...
+void send_result_message_BT(int device_num, int x, int y, int gamestatus, int hitstatus, int destroyed_start_x, int destroyed_start_y, int length, int orientation, bool single_player_mode) {
     stringstream message;
     message << "result " << x << " " << y << " " << gamestatus << " " << hitstatus << " " << destroyed_start_x << " " << destroyed_start_y << " " << length << " " << orientation << "~";
 
     if (device_num == 1) {
         BT_send_0(message.str().c_str());
     }
     else {
-        BT_send_1(message.str().c_str());
+        if (!single_player_mode) {
```
Comment (paullin03) [inline software/gamelogic/inputs.cpp]: else-if instead

**Review by paullin03 — APPROVED** [review]

File: software/gamelogic/gamelogic.cpp (hunk 1 above)
Comment (neotsai0622) [reply software/gamelogic/gamelogic.cpp:61]: I don't have an answer to this. Have to ask @Eleiah or @mslongo 
my guess is not though from the brief look of the function

File: software/gamelogic/gamelogic.cpp (hunk 2 above)
Comment (neotsai0622) [reply software/gamelogic/gamelogic.cpp]: done

File: software/gamelogic/inputs.cpp (hunk 3 above)
Comment (neotsai0622) [reply software/gamelogic/inputs.cpp]: done

File: software/gamelogic/inputs.cpp (hunk 4 above)
Comment (neotsai0622) [reply software/gamelogic/inputs.cpp]: done

File: software/gamelogic/gamelogic.cpp (hunk 1 above)
Comment (Eleiah) [reply software/gamelogic/gamelogic.cpp:61]: I think I have tested Init_RS232() with only one card. It still worked. I could just move the card manually and the other port would work.
If this isn't what you're talking about then idk @paullin03

**Pull request merged**